modules/widgets/eu-cookie-law.php

```diff
@@ -156,6 +159,11 @@ public function widget( $args, $instance ) {
 
 			$instance = wp_parse_args( $instance, $this->defaults() );
 
+			if ( Jetpack_AMP_Support::is_amp_request() ) {
```

Comment:
```suggestion
			if ( class_exists( 'Jetpack_AMP_Support' ) && Jetpack_AMP_Support::is_amp_request() ) {
```

Reply:
@jeherve This has been fixed in the latest commit.